spec.md

```diff
@@ -1793,6 +1843,42 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on the node.
```

Comment:
It's unclear to me when CO needs to call this. For instance, should the CO call this when the volume is in `NODE_READY` state (i.e., after `ControllerPublishVolume` is successful)? or in `PUBLISHED` state (i.e., `NodePublishVolume` is successful)?

Reply:
A follow-up question: If `NodeExpandVolume` should be called after `NodeStageVolume` or `NodePublishVolume`, then say if the volume is multi-node capable and has been staged or published on multiple nodes, can the effect of calling `NodeExpandVolume` on one node be seen on all other nodes?

Reply:
@jieyu  Yeah I think some clarification is in order for when the CO should call this. I was thinking rather than prescribing when a CO can call this, if we made it clear that `NodeExpandVolume` does not implicitly call `NodeStageVolume` or `NodePublishVolume` and call to `NodeExpandVolume` expects volume to be available in specified `volume_path` then CO will have enough information to decide when to make this call.

For example - for something like `EBS`, the CO will call this after `NodeStageVolume` but most likely before `NodePublishVolume`. I do not feel good about being very specific in call order and I think as long as CO knows expectation of this function, he can decide.

Reply:
@chhsia0 That depends on file system. For some file systems like `gfs2`, growing the file system on one node typically makes the space available on all nodes. 

But I think that any fencing that needs to be implemented to support file system types that can be mounted as rw on multiple nodes has to be implemented by CO. In k8s for example, this could mean that annotating a PVC that is RW mounted on multiple nodes so as only one node can expand the file system.

Reply:
@jieyu @chhsia0 expanded docs with flow about when this can be called.